inference-engine/src/vpu/graph_transformer/src/frontend/frontend.cpp

```diff
@@ -21,9 +21,9 @@
 
 #include <legacy/convert_function_to_cnn_network.hpp>
 #include <generic_ie.hpp>
+#include <ngraph/pass/manager.hpp>
```

Comment:
Is it because this header has been included in `apply_transformations_to_ti_body` or we can remove it?

Reply:
Right.